infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -150,7 +150,6 @@ public void prefixTrim(@Nonnull long address, @Nonnull long epoch) {
     public void waitForEpochWaterMark(@Nonnull long epoch) {
```

Comment:
please delete `@Nonnull` it's useless for primitives

Reply:
Done.